addon-test-support/index.ts

```diff
@@ -12,5 +12,7 @@ export {
   middlewareReporter as _middlewareReporter,
   setupMiddlewareReporter,
 } from './setup-middleware-reporter';
+export { storeResults, printResults } from './logger';
+export { setupConsoleLogger } from './setup-console-logger';
```

Comment:
Just a note: we should queue up a subsequent conversation RE: providing a generic `import { setup } from 'ember-a11y-testing'` type API that internally calls all of the methods we are exporting. My idea here is to make it roughly similar to the things we are doing in ember-qunit's `start` (where each thing it does can be opted out of, and controlled manually by dropping down a layer as needed).

https://github.com/emberjs/ember-qunit/blob/1d3decdd60b78bc07a4bc4c20f53075de6467600/addon-test-support/index.js#L175-L230

AFAICT, we have all the required pieces, and it would be easy to add in a follow up.

Reply:
Yep, I like this idea, and agree.